spec/cassette/client/cache_spec.rb

```diff
@@ -22,14 +18,13 @@
   describe '.backend=' do
     it 'sets the cache' do
       # setup
-      global_cache = double('cache_store')
-      logger = Logger.new('/dev/null')
+      global_cache = instance_double(described_class, 'cache_store')
```

Comment:
If you want to be thorough, global cache is a [ActiveSupport::Cache::Store](https://api.rubyonrails.org/classes/ActiveSupport/Cache/Store.html) and not a Cassette::Client::Cache